.gitignore

```diff
@@ -23,3 +23,5 @@ pa11y-report.html
 
 # Next export
 out/
+
+.DS_Store
```

Comment:
Keeping the gitignore specific to the project keeps it clean, shouldn't these things live in our global gitignore?

(says while knowing he checked it in)

Reply:
haha – well it is in _my_  global .gitignore

Don't think it does any harm to be in here, though?